src/Controls/src/Core/ContentView.cs

```diff
@@ -21,12 +21,15 @@ protected override void OnBindingContextChanged()
 		{
 			base.OnBindingContextChanged();
 
-			View content = Content;
+			IView content = Content;
+
+			if (content == null && (this as IContentView)?.PresentedContent is IView presentedContent)
+				content = presentedContent;
+
 			ControlTemplate controlTemplate = ControlTemplate;
```

Comment:
I see templates here... do we have a test for templated and non-templated controls? Just want to make sure that we catch everything.